lib/passpartu/verify.rb

```diff
@@ -37,7 +45,21 @@ def role_ignore?
     end
 
     def check_policy
-      @result = Passpartu.policy.dig(role, *keys)
+      @result = waterfall_rules? ? check_waterfall_rules : simple_check
+    end
+
+    def simple_check
```

Comment:
What is it?
and what the difference between  `check_waterfall_rules`

Reply:
simple_check need when waterfall_rules is false, so it don't check waterfall